tests/cupyx_tests/scipy_tests/ndimage_tests/test_interpolation.py

```diff
@@ -940,6 +940,20 @@ def test_spline_filter1d_output(self, xp, scp, array_order):
         return output
 
 
+# See #5537
+@testing.with_requires('scipy')
+class TestSplineFilter1dLargeArray:
```

Comment:
Do we need `@testing.slow`?

Reply:
Yep, let me mark `@testing.slow`. This test took about three minutes to complete all modes on my local.